lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -32,6 +32,7 @@ define((require, exports, module) ->
           stack.push(pushedState)
         return pushedState
 
+    #The same as pushState, but switch our finite state to "to" state
```

Comment:
with the purpose of what? Comment doesn't need to rephrase hwat is written in the code, we see that the difference is that "to" is returned instead of "pushedState". What's the purpose of this change? When shall Icall pushState and when shall I call pushStateCheckout?